From e7b65d310d006dcff49d6c378bb501862683852b Mon Sep 17 00:00:00 2001
From: Colin Guthrie <colin@mageia.org>
Date: Sun, 4 Sep 2011 12:28:07 +0200
Subject: [PATCH 110/111] stream-restore: Add in some variable sets that were
 missing from 9ffa93.

This broke the conversion of the stream-restore database from
previous versions.
---
 src/modules/module-stream-restore.c |    8 ++++++++
 1 files changed, 8 insertions(+), 0 deletions(-)

diff --git a/src/modules/module-stream-restore.c b/src/modules/module-stream-restore.c
index 506c5fc..22659df 100644
--- a/src/modules/module-stream-restore.c
+++ b/src/modules/module-stream-restore.c
@@ -1078,6 +1078,14 @@ static struct entry* legacy_entry_read(struct userdata *u, pa_datum *data) {
     }
 
     e = entry_new();
+    e->muted_valid = le->muted_valid;
+    e->muted = le->muted;
+    e->volume_valid = le->volume_valid;
+    e->channel_map = le->channel_map;
+    e->volume = le->volume;
+    e->device_valid = le->device_valid;
+    e->device = pa_xstrdup(le->device);
+    e->card_valid = le->card_valid;
     e->card = pa_xstrdup(le->card);
     return e;
 }
-- 
1.7.5.4

